instrumentation/spring/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -419,8 +419,8 @@
     },
     {
       "name": "otel.instrumentation.logback-appender.experimental.capture-mdc-attributes",
-      "type": "java.util.List<java.lang.String>",
-      "description": "MDC attributes to capture. Use the wildcard character <code>*</code> to capture all attributes."
+      "type": "java.lang.String",
```

Comment:
@jeanbisutti this property is used by https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/5b21872533968c9fdb0347b3a8526f84aed19174/instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/internal/instrumentation/logging/LogbackAppenderInstaller.java#L19 - and I don't know if we can access `@CofigurationProperties` at that stage yet

Reply:
Not sure what you mean.

From what I observe, `@Value` does not seem to work in `LogbackAppenderApplicationListener`. 

`org.springframework.boot.context.logging.LoggingApplicationListener` does not use `@Value` .